### PR TITLE
JWT-2107 Added fonts in new formats (WOFF, WOFF2)

### PR DESCRIPTION
**Pull request by werthdavid**
Als Vorarbeit für JWT-2107 wurden hier die Fonts in WOFF und WOFF2 zusätzlich abgelegt

### CODE REVIEW

**Comment by theMattCode** [comment]
The PNGs with actual and expected results shouldn't be included in the commit, should they?

**Comment by werthdavid** [comment]
> The PNGs with actual and expected results shouldn't be included in the commit, should they?

Thanks Matthäus, in fact the whole folder should not be part of the repo

**Review by drewshz — APPROVED** [review]

**Review by hennejg — APPROVED** [review]

File: .github/workflows/continuous-delivery.yml [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ steps.create_release.outputs.upload_url }} # This pulls from the CREATE RELEASE step above, referencing it's ID to get its outputs object, which include a `upload_url`. See this blog post for more info: https://jasonet.co/posts/new-features-of-github-actions/#passing-data-to-future-steps
+          asset_path: ./target/documentplatform-standard14fonts-${{ steps.semanticversion.outputs.new_version }}.jar
+          asset_name: documentplatform-standard14fonts-${{ steps.semanticversion.outputs.new_version }}.jar
+          asset_content_type: application/java-archive
+
+      - name: Upload Release Asset - sources jar
+        id: upload-release-asset
```
Comment (benjaminalbanese) [inline .github/workflows/continuous-delivery.yml:114]: so far we only uploaded one asset per release - no idea if that stunt here is really working. 

If not, plan B would be to trigger a build after the release like doing so in our web toolkit project with tomcat 9 + java 11. In the new run, build a matrix with the 3 suffixes "", "javadoc", "sources" and try to upload all 3 therefore in different runs.

Comment (benjaminalbanese) [reply .github/workflows/continuous-delivery.yml]: Some smaller bugs detected in the playground - project, which should be fixed now. 

@werthdavid  if you want, imho you can merge the branch now if you want.

**Comment by werthdavid** [comment]
Instead of placing WOFF and WOFF2 files inside the JAR I made a method in JWT that can convert any TTF font to WOFF/WOFF2 and not only standard14.

**Pull request closed without merging**